acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -0,0 +1,86 @@
+test_name 'C98120, C98077: Sensitive Data is redacted on CLI, logs, reports' do
+require 'puppet/acceptance/puppet_type_test_tools.rb'
+extend Puppet::Acceptance::PuppetTypeTestTools
+
+  app_type        = File.basename(__FILE__, '.*')
+  tmp_environment = mk_tmp_environment_with_teardown(master, app_type)
+  fq_tmp_environmentpath  = "#{environmentpath}/#{tmp_environment}"
+
+  tmp_filename = "/tmp/sensitive_content_#{random_string}.txt"
```

Comment:
Unfortunately, this path is not valid on Windows. Either the `generate_` commands need to be aware of the context of the target agent, or the resource hashes themselves need to include conditionals for `osfamily`.